### PR TITLE
#121: Added colors to math buttons palette

### PR DESCRIPTION
**Pull request by oskarhinc**
screenshot from 2018-06-18 16-24-55

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -244,6 +243,26 @@ a {
 	background-color: #fafafa;
 }
 
+.btn-delete {
+	background-color: #f0f0f0!important;
```
Comment (pgesek) [inline styles/styles.css]: any particular reason for he !important part?

**Comment by pgesek** [comment]
@oskarhinc prefix the commit/PR with #<ticket-number>

File: styles/styles.css (hunk 1 above)
Comment (oskarhinc) [reply styles/styles.css]: No, it's not needed. I've forgotten to remove it. Thanks

**Comment by pgesek** [comment]
@oskarhinc looks good, note that you still need t add the background styling + plus the grouping of buttons in the screenshot is laid out a bit different.

@johnhbenetech @sueannma I assume that the grouping of buttons in #121 is intentional?

**Comment by johnhbenetech** [comment]
@oskarhinc @pgesek that's correct. grouping should be as in #121. We can discuss on call if we want to break this task down into smaller parts

**Comment by oskarhinc** [comment]
@pgesek You're right, the background is still to do. 
@johnhbenetech Maybe I'll leave you the questions about grouping below #121

**Pull request merged**